pkg/commands/logging/bigquery/bigquery_test.go

```diff
@@ -126,7 +126,7 @@ func TestUpdateBigQueryInput(t *testing.T) {
 			want: &fastly.UpdateBigQueryInput{
 				ServiceID:         "123",
 				ServiceVersion:    4,
-				Name:              "log",
+				Name:              *fastly.String("log"),
```

Comment:
I get the sense there was some automation script running here 🙂 

I don't think we need to wrap a string in a pointer and then dereference the pointer 😄 

If you could look out and correct these overlaps.

Reply:
Oh dear! Fixed in https://github.com/fastly/cli/pull/707/commits/a8210d4bb2b90b9427b452f229817faa14326835